auth-web/src/components/auth/manage-business/AddBusinessDialog.vue

```diff
@@ -191,6 +218,22 @@ export default class AddBusinessDialog extends Vue {
     return CommonUtils.isFirmNumber(this.businessIdentifier)
   }
 
+  get showCertifier (): boolean {
+    return this.isFirm && this.isGovStaffAccount
+  }
+
+  get currentUserName (): string {
+    if (this.isGovStaffAccount) return this.certifiedByName
+    return `${this.userLastName}, ${this.userFirstName}`
+  }
+
+  get certifiedByNameRules (): any[] {
+    return [
+      v => !!v || 'Certification is required',
+      v => v.length <= 150 || 'Maximum 150 characters'
```

Comment:
I see this is code copied from the passcode. However, I think you don't need this line since maxLength is already being used.

Reply:
The text is different in passcodeRules, so I have to use another rule.

Reply:
I was not saying to reuse the passcode rules! I was saying that you don't need line 233.

Reply:
Sure. I can remove line 233.